static/bidder-params/synacormedia.json

```diff
@@ -8,8 +8,12 @@
     "seatId": {
       "type": "string",
       "description": "The seat id."
+    },
+    "tagId": {
+      "type": "string",
+      "description": "The tag id."
     }
   },
 
-  "required": ["seatId"]
+  "required": ["seatId", "tagId"]
```

Comment:
Do you have any current traffic coming from PBS? When you set `tagId` as required, if a request comes in missing `tagId` once it is set to required, it will mark the whole request as failed and not send a bid request to any bidder. So adding a required field can break the entire prebid setup of a publisher.

This behavior is to help ensure publishers set up their configs correctly. Otherwise a bidder might set up a request with 5 bidders, but bidder ABC has a bad config and always errors out. If the other 4 are working normally, the publisher might not notice they have a bad config for quite a while. If some pages have a valid ABC config, they might just conclude that ABC just doesn't bid much if they don't troubleshoot carefully.

If there are publishers who currently have configs that omit `tagId`. we would recommend that initially the adapter not advertise `tagId` as required in `bidder-params`, but return an error from within the adapter if it is missing. An error from within the adapter will not invalid the whole request, allowing the other bidders to function. Then after all publishers have had a reasonable amount of time to update their configs, we can do the additional step of making it required in `bidder-params`.

Reply:
I'm not sure who is using this tag, so I went ahead and removed tagId from the required params.